src/main/java/edu/cornell/kfs/module/purap/jaggaer/supplier/xml/Discount.java

```diff
@@ -0,0 +1,54 @@
+package edu.cornell.kfs.module.purap.jaggaer.supplier.xml;
+
+import java.util.ArrayList;
+import java.util.List;
+import jakarta.xml.bind.annotation.XmlAccessType;
+import jakarta.xml.bind.annotation.XmlAccessorType;
+import jakarta.xml.bind.annotation.XmlAttribute;
+import jakarta.xml.bind.annotation.XmlElement;
+import jakarta.xml.bind.annotation.XmlElements;
+import jakarta.xml.bind.annotation.XmlRootElement;
+import jakarta.xml.bind.annotation.XmlType;
+import jakarta.xml.bind.annotation.adapters.CollapsedStringAdapter;
+import jakarta.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "", propOrder = { "discountPercentOrDiscountAmountOrIsoCurrencyCode" })
+@XmlRootElement(name = "Discount")
+public class Discount {
+
+    @XmlAttribute(name = "isChanged")
+    @XmlJavaTypeAdapter(CollapsedStringAdapter.class)
+    protected String isChanged;
+    @XmlAttribute(name = "unit", required = true)
+    @XmlJavaTypeAdapter(CollapsedStringAdapter.class)
+    protected String unit;
+    @XmlElements({ @XmlElement(name = "DiscountPercent", required = true, type = DiscountPercent.class),
+            @XmlElement(name = "DiscountAmount", required = true, type = DiscountAmount.class),
+            @XmlElement(name = "IsoCurrencyCode", required = true, type = IsoCurrencyCode.class) })
+    protected List<Object> discountPercentOrDiscountAmountOrIsoCurrencyCode;
```

Comment:
that is a pretty long variable name, could we use a shorter name?

Reply:
ok, I have shortened the variable name, and added an interface